resources/web/wwi/nodes/utils/WbMatrix4.js

```diff
@@ -0,0 +1,111 @@
+import WbMatrix3 from './WbMatrix3.js';
+import WbVector4 from './WbVector4.js';
+
+export default class WbMatrix4 {
+  constructor() {
+    this.m = [1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0];
+  }
+
+  extracted3x3Matrix() {
+    return new WbMatrix3(this.m[0], this.m[1], this.m[2], this.m[4], this.m[5], this.m[6], this.m[8], this.m[9], this.m[10]);
+  }
+
+  inverse() {
+    const inv = [];
```

Comment:
```suggestion
    const inv = new Array(16);
```
I am surprised it was working before. This should be slightly faster.

Reply:
I don't like this syntax too much because I find it error prone. `new Array(16)` will effectively create an array of 16 but `new Array('16')` will create an array of length 1 with 16 as element. But yes it should slightly improve the performance so I will apply it.